DelfinAnomalyDetection/Readme.md

```diff
@@ -0,0 +1,80 @@
+The example is POC of Anomaly detection of delfin metric data
+Kafka is used data stream for receiving metric data which is plugged in into Anomaly detection module.
+
+### Architecture
+
+````
+ ---------            ---------            ------------------
+| Delfin  | ------>  |  Kafka  |  ----->  | Anomaly Detection |
+ ---------            ---------            -------------------
+````
+
+- Delfin publish metric data on kafka (Default topic: delfin-kafka)
+- Anomaly detection module subscribe for the topic and store the metric data in dictionary
+- Anomaly detections runs on stored data in dictionary
+
+### Constraints
+
+- Currently only "read_bandwidth" considered
+- (TODO) Online model training is not implemented 
+- Only 1000 samples are considered for model training
+
+### Steps:
+- Setup Delfin and Kafka with help of [documentation](https://docs.sodafoundation.io/guides/developer-guides/delfin/)
+- install dependencies mentioned in requirements.txt ```pip install -r requirements.txt```
+- export PYTHONPATH=`pwd`
```

Comment:
Hope the steps can be little more detail , like
git clone or download repo
cd /<delfinanalomalidetection> etc??

Reply:
added steps for git clone and stepping into directory